src/main/resources/db/migration/V42__add_oic_hearing_id_to_hearings.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE hearing ADD COLUMN oic_hearing_id bigint  not null;
```

Comment:
If we add `not null` constraint to a table with existing records, will this fail when Flyway runs it? Do we need to provide `default`?

Reply:
so id rather  not add a default.  My plan was to delete any records in the dev db, as its dev.